doc/chapter-configuration.asciidoc

```diff
@@ -30,6 +30,22 @@ unbind-key       R
 bind-key         ^R    reload-all
 ----
 
+=== Splitting macros into multiple lines
```

Comment:
Macros is just one example where this feature could be useful. How about rewriting this to say something like "Splitting long lines into multiple ones"?